render/template.go

```diff
@@ -67,6 +74,60 @@ func (s templateRenderer) exec(name string, data Data) (template.HTML, error) {
 	return template.HTML(body), nil
 }
 
+func (s templateRenderer) addAssetsHelpers(helpers map[string]interface{}) map[string]interface{} {
+	helpers["assetPath"] = func(file string) template.HTML {
+		return template.HTML(s.assetPath(file))
+	}
+
+	helpers["javascriptTag"] = func(file string) template.HTML {
+		jsTag := tags.New("script", tags.Options{
+			"type": "text/javascript",
+			"src":  s.assetPath(file),
+		})
+
+		return jsTag.HTML()
+	}
+
+	helpers["stylesheetTag"] = func(file string) template.HTML {
+		cssTag := tags.New("link", tags.Options{
+			"rel":   "stylesheet",
+			"media": "screen",
+			"href":  s.assetPath(file),
+		})
+
+		return cssTag.HTML()
+	}
+
+	return helpers
+}
+
+func (s templateRenderer) assetPath(file string) string {
+	manifest, err := s.AssetsBox.MustString("manifest.json")
```

Comment:
I feel like we might want to do some sort of caching here if not in `development` mode.